orchestrator-salt/src/main/resources/salt/salt/kerberos/scripts/principals.sh

```diff
@@ -22,5 +27,11 @@ else
   done
 fi
 
-salt-cp -G 'roles:kerberos_server_slave' /etc/krb5.keytab /etc/krb5.keytab
-salt-cp -G 'roles:kerberos_server_slave' /var/kerberos/krb5kdc/.k5.{{ realm }} /var/kerberos/krb5kdc/.k5.{{ realm }}
+if [ -x "$(command -v activate_salt_env)" ];
+then
+  . activate_salt_env
+  salt_copy_kerberos_files
```

Comment:
what happens if "salt_copy_kerberos_files" fails? Is it possible that salt's python env stays active for other scripts?

Reply:
as far as i know modified env. variables will consist only in the context of this script.  thus the virtualenv should become deactivated at the end of the script. correct me, if i'm wrong.